Plugins/BuildServerIntegration/AppVeyorIntegration/AppVeyorAdapter.cs

```diff
@@ -458,7 +473,7 @@ private Task<Stream> GetStreamFromHttpResponseAsync(HttpClient httpClient, Task<
                 return task.CompletedResult().Content.ReadAsStreamAsync();
             }
 
-            return null;
+            return null; // TODO NULLABLE returning a null Task is a no no
```

Comment:
Should this be `Task.FromResult(null)`?

Reply:
Looks like it. Requires some other changes.